interface/src/app/utils.spec.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Unit tests.
+ */
+
+// Test the hyperGeometricTest function
+describe('Test for hypergeometric test function in utils.js', function() {
+  beforeEach(module('adage.utils'));
+
+  var mockService;
+
+  beforeEach(function() {
+    angular.mock.inject(function($injector) {
+      mockService = $injector.get('MathFuncts');
+    });
+  });
+
+  it('should be defined', function() {
+    expect(mockService).toBeDefined();
+  });
+
+  // The following functions will make sure that the output for the
+  // hyperGeometricTest function matches the output produced by
+  // http://stattrek.com/m/online-calculator/hypergeometric.aspx and
+  // https://www.geneprof.org/GeneProf/tools/hypergeometric.jsp for
+  // some given k values.
+  it('should output a desired p-value for k=1', function() {
+    expect(+(1 - mockService.hyperGeometricTest(1, 5, 10, 50)).toFixed(10))
+    .toEqual(0.689437218);
+  });
+
+  it('should output a desired p-value for k=2', function() {
```

Comment:
Yep, good idea. In fact, I will just go ahead and put all of these 5 into one ``it()`` call to make the code clean and concise, while still making sure that multiple calls of the service do not affect one another.